packages/jasmine/src/jasmine_runner.js

```diff
@@ -185,6 +184,12 @@ function main(args) {
     }
   }
 
+  // If the user provides a seed as an environment variable, configure jasmine to use it.
+  const envSeed = process.env['JASMINE_SEED'];
```

Comment:
I'm surprised that this works. Bazel generally strips out all the environment variables to keep actions hermetic - you can imagine if some environment variable doesn't have the same value on your machine as your coworker, you wouldn't get any remote cache hits since the cache key is determined from all the inputs.

https://bazelbuild.github.io/rules_nodejs/node/node.html#nodejs_test lists the `configuration_env_vars` attribute which is an allowlist for some environment variables to be passed through

Reply:
You're right, I've changed my approach and tested locally.

Now it's allowing an argument to be passed for `seed`. Documentation updated accordingly.